inductiva/scenarios.py

```diff
@@ -16,14 +15,13 @@ class Scenario(ABC):
     """Base class for scenarios."""
     valid_simulators = []
 
-    def _setup_output_dir(self, output_dir: str):
+    def _output_dir_name(self, output_dir: str):
```

Comment:
Any reason to remove the time_stamp and resolve_path?

Reply:
I am removing them from here, because the same code is duplicated in `simulator` class, so the timestamps and resolve path are added there not here.

Reply:
Top!